pkgs/development/tcl-modules/by-name/ji/jimlib/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  fetchFromGitLab,
+  mkTclDerivation,
+}:
+
+mkTclDerivation rec {
+  pname = "jimlib";
+  version = "0.16.0";
+
+  src = fetchFromGitLab {
+    owner = "dbohdan";
+    repo = "jimlib";
+    rev = "v${version}";
+    hash = "sha256-a7Su/JByGLgnYBCRbP9z9CunTIYTwAisWuQafZWYgYo=";
+  };
+
+  dontBuild = true;
+  doCheck = true;
+
+  checkPhase = ''
+    runHook preCheck
+    tclsh ./jimlib.tcl
+    runHook postCheck
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm644 jimlib.tcl $out/lib/jimlib/jimlib.tcl
+    install -Dm644 pkgIndex.tcl $out/lib/jimlib/pkgIndex.tcl
+    runHook postInstall
+  '';
+
+  meta = {
+    homepage = "https://gitlab.com/dbohdan/jimlib/";
+    description = "Personal standard library for Jim Tcl";
+    license = [
+      lib.licenses.mit
+      lib.licenses.tcltk
+      lib.licenses.bsd3
+    ];
+    platforms = lib.platforms.all;
```

Comment:
Are you sure? This should probably be the platforms Tcl supports

Reply:
It depends on (jim)tcl, so it's going to be restricted to its platforms anyways, and jimlib itself doesn't look platform-dependent to me (except for a few commands), so I think `all` is fine

Reply:
Source has mentions of bsd and windows and the tcl interpreter has platforms set to all as well